backend/src/test/java/com/woowacourse/levellog/acceptance/PreQuestionAcceptanceTest.java

```diff
@@ -0,0 +1,201 @@
+import static org.hamcrest.Matchers.notNullValue;
+import static org.springframework.restdocs.restassured3.RestAssuredRestDocumentation.document;
+
+import com.woowacourse.levellog.fixture.RestAssuredResponse;
+import com.woowacourse.levellog.levellog.dto.LevellogDto;
+import com.woowacourse.levellog.prequestion.dto.PreQuestionDto;
+import com.woowacourse.levellog.team.dto.ParticipantIdsDto;
+import com.woowacourse.levellog.team.dto.TeamCreateDto;
+import io.restassured.RestAssured;
+import io.restassured.response.ValidatableResponse;
+import java.time.LocalDateTime;
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+
+@DisplayName("사전 질문 관련 기능")
+public class PreQuestionAcceptanceTest extends AcceptanceTest {
+
+    /*
+     * Scenario: 사전 질문 등록하기
+     *   given: 1개의 레벨로그와 2명의 사용자(페퍼, 이브)가 있다.
+     *   when: 이브가 페퍼의 레벨로그에 사전 질문 등록을 요청한다.
+     *   then: 201 Created 상태 코드와 Location 헤더에 /api/levellogs/{levellogsId}/pre-question/{preQuestionId}를 담아 응답받는다.
+     */
+    @Test
+    @DisplayName("사전 질문 등록하기")
+    void create() {
+        // given
+        final RestAssuredResponse pepperResponse = login("페퍼");
+        final RestAssuredResponse eveResponse = login("이브");
+        final String pepperToken = pepperResponse.getToken();
+        final String eveToken = eveResponse.getToken();
+        final Long eveMemberId = eveResponse.getMemberId();
+
+        final TeamCreateDto teamRequest = new TeamCreateDto("잠실 제이슨조", "트랙룸", 1, LocalDateTime.now().plusDays(3),
+                new ParticipantIdsDto(List.of(eveMemberId)));
+        final String teamId = post("/api/teams", pepperToken, teamRequest).getTeamId();
+        final LevellogDto levellogRequest = LevellogDto.from("페퍼의 레벨로그");
+        final String levellogId = post("/api/teams/" + teamId + "/levellogs", pepperToken,
+                levellogRequest).getLevellogId();
+
+        final PreQuestionDto request = new PreQuestionDto("이브가 쓴 사전 질문");
+
+        // when
+        final ValidatableResponse response = RestAssured.given(specification).log().all()
+                .header(HttpHeaders.AUTHORIZATION, "Bearer " + eveToken)
+                .body(request)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .filter(document("pre-question/create"))
+                .when()
+                .post("/api/levellogs/" + levellogId + "/pre-questions")
+                .then().log().all();
+
+        // then
+        response.statusCode(HttpStatus.CREATED.value())
+                .header(HttpHeaders.LOCATION, notNullValue());
+    }
+
+    /*
+     * Scenario: 사전 질문 수정하기
+     *   given: 1개의 레벨로그와 2명의 사용자(페퍼, 이브)가 있다.
+     *   given: 이브가 페퍼에게 작성한 사전 질문이 있다.
+     *   when: 이브가 페퍼에게 쓴 레벨로그의 사전 질문 수정을 요청한다.
+     *   then: 204 No Content 상태 코드를 응답받는다.
+     */
```

Comment:
```suggestion
    /*
     * Scenario: 사전 질문 수정하기
     *   given: 1개의 레벨로그와 2명의 사용자(페퍼, 이브)가 있다.
     *   given: 이브가 페퍼에게 작성한 사전 질문이 있다.
     *   when: 이브는 페퍼의 레벨로그에 쓴 사전 질문 수정을 요청한다.
     *   then: 204 No Content 상태 코드를 응답받는다.
     */
```
이게 좀 더 명확하지 않을까요...?